sys/include/net/ng_lomac.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    sys_net_ng_lomac    Loopback MAC layer
+ * @ingroup     sys_net
+ * @{
+ *
+ * @file
+ * @brief       Interface definition for the loopback MAC layer
+ *
+ * @details     This MAC layer triggers a receive command everytime a
+ *              packet is send to it.
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+
+
+#ifndef NG_LOMAC_H_
+#define NG_LOMAC_H_
+
+#include "kernel_types.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Default message queue size for loopback MAC layer.
+ */
+#ifndef NG_LOMAC_MSG_QUEUE_SIZE
+#define NG_LOMAC_MSG_QUEUE_SIZE (8U)
+#endif
+
+/**
+ * @brief   The PID to the loopback MAC layer thread.
+ */
+extern kernel_pid_t lomac_pid;
```

Comment:
As long as the mtu is configured to be greater than 1280 bytes, there shouldn't be any problems. There is no statement about an upper limit of the mtu. So in general it could be plausible to set this to `UINT16_MAX`. But I guess this is completely dependent on the underlying architecture.

>IPv6 requires that every link in the internet have an MTU of 1280
   octets or greater.  On any link that cannot convey a 1280-octet
   packet in one piece, link-specific fragmentation and reassembly must
   be provided at a layer below IPv6.